src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -196,12 +196,50 @@ private void LookupAllExtensionMembersInSingleBinder(LookupResult result, string
             singleLookupResults.Free();
         }
 
+        private void LookupAllNewExtensionMembersInSingleBinder(LookupResult result, string? name,
```

Comment:
>NewExtensionMembers [](http://example.com/codeflow?start=30&length=19)

Similar comment for "NewExtensionMembers" term. It looks like this is applicable to other new APIs.